go.mod

```diff
@@ -4,7 +4,7 @@ go 1.20
 
 require (
 	github.com/agext/levenshtein v1.2.3 // indirect
-	github.com/cloudflare/cloudflare-go v0.87.0
+	github.com/cloudflare/cloudflare-go v0.87.1-0.20240205234129-cca23db4beb9
```

Comment:
Once new golang library is released i can scrap these changes

Reply:
please scrap them before hand. we can test locally by shimming in the newer version without potentially causing merge conflicts.